app/Http/Requests/Three/PostRequest.php

```diff
@@ -26,6 +26,7 @@ public function rules()
         return [
             'status' => 'in:pending,accepted,rejected',
             'caption' => 'string|max:140',
```

Comment:
should we make caption required here?

Reply:
I think that makes sense (at least for how things work now). 👍

Reply:
oh and also make quantity required?

Reply:
quantity should not be required since PN won't be asking for it often.

Reply:
hmm @DFurnes when I made the caption required and tested to not sent, it seems like the error handling was off again and broke with a `500` response but didn't have anything in the body. Should we hold off on that until we fix the error handling in Rogue (also made a new card for that [here](https://www.pivotaltracker.com/n/projects/2019429/stories/153458427)!).